### ISSUE TITLE
Add dropout layer to a Resnet Model

### ISSUE THREAD

**Issue by abdou31** [tensorflow/tensorflow#33940]
Hello, 
This is not a bug, I'm searching for help or a suggestion, I have posted many questions on StackOverflow but I didn't get any answer.
I search for a way to add a dropout layer to improve the precision and accuracy of the model.
```python
    flatten = tf.layers.flatten(inputs=conv8)

   # dropout=tf.layers.dropout(inputs=flatten,training=false) # Add dropout here

    # Dense layer 1, a fully connected layer.
    dense1 = tf.layers.dense(
        inputs=flatten,
        units=1024,
        activation=tf.nn.relu,
        use_bias=True)

    # Dense layer 2, also known as the output layer.
    logits = tf.layers.dense(
        inputs=dense1,
        units=80,
        activation=None,
        use_bias=True,
        name="logits")

    # Make prediction for PREDICATION mode.
    predictions_dict = {
        "name": features['name'],
        "logits": logits
    }

```

Is it possible to add dropout layer between the flatten and the dense layer?

**Comment by ymodak** [comment]
The most common practice is to use dropout layer after fully connected layer (Dense).
See [Dropout: A Simple Way to Prevent Neural Networks from Overfitting](http://jmlr.org/papers/v15/srivastava14a.html)
 However you may try using dropout between flatten and dense layer and study the effects.

**Comment by abdou31** [comment]
Already, I have trying using between the flatten and dense layer.
+ No change in results and graph of the loss.

I will try using the dropout layer after the dense layer ( dense 1 ) and I will tell you what is changed.

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
